product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/custom_monitoring/index.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: "Custom Monitoring"
+---
+
+BigAnimal has the ability to expose metrics and logs. To enable this on your cluster:
+
+TBC - will need input from UI team
+
+## Metrics
+
+If you are using `customer metrics` you will be able to access metrics in a prometheus format. We have provided some [example metrics](/biganimal/latest/using_cluster/05_monitoring_and_logging/metrics/#cnp_pg_replication) to help get you started.
```

Comment:
I think you meant to link to the child example metrics topic, correct?
```suggestion
If you are using `customer metrics` you will be able to access metrics in a prometheus format. We have provided some [example metrics](/biganimal/latest/using_cluster/05_monitoring_and_logging/custom_monitoring/example_metrics/) to help get you started.
```

Reply:
Yep, I have merged your suggestion